src/map-app/app/view/sidebar.ts

```diff
@@ -125,7 +120,7 @@ export class SidebarView extends BaseView {
     d3.select("#map-app-sidebar i").attr("class", "fa fa-angle-left");
 
     this.presenter.changeSidebar(); // Refresh the content of the sidebar
-    if (document.getElementById("dir-filter") && window.outerWidth >= 1080)
+    if (document.getElementById("dir-filter") && getViewportWidth() > 1080)
```

Comment:
Ditto; although I see here perhaps `getViewportWidth()` computes something else.  Wondering, why was `outerWidth` not working adequately?

Reply:
Outerwidth includes other browser layout, like the scrollbar and dev tools. Inner width is more accurate but not exactly the same as @media in css, so this is just to make it consistent.